vta/include/vta/sim_tlpp.h

```diff
@@ -0,0 +1,162 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
```

Comment:
As a side note, given this interface is not part of public interface used by the user, let us move it to the internal header, as opposed to keep it in the include folder